### DISCUSSION TITLE
Using RHF in IE11

### DISCUSSION THREAD

**Discussion by c0d3x** [react-hook-form/react-hook-form#2930] [Q&A]
I get this error:
![image](https://user-images.githubusercontent.com/661911/93334123-ae007c80-f824-11ea-8e4f-dbd127a7939b.png)
![image](https://user-images.githubusercontent.com/661911/93334167-bfe21f80-f824-11ea-8dd9-564ef340eea7.png)

Anyone know what to do? I included these two:
import { useForm } from 'react-hook-form/dist/index.ie11';
import { yupResolver } from '@hookform/resolvers/dist/index.ie11';

**Comment by bluebill1049** [comment]
oh wow, IE 11...

**Reply by bluebill1049** [reply]
i have no idea this one, anyone can help?
